lib/timeline/component/Group.js

```diff
@@ -498,7 +496,7 @@ Group.prototype._updateSubgroupsSizes = function () {
       })
 
       me.subgroups[subgroup].start = newStart;
-      me.subgroups[subgroup].end = newEnd;
+      me.subgroups[subgroup].end = new Date(newEnd - 1) // -1 to compensate for colliding end to start subgroups;
```

Comment:
The reason I do want to leave the `new Date()` is for consistency. I don't want the `start` to be a Date and the `end` to be a Date in milliseconds.